help/getting-started/adobe-ims-config.md

```diff
@@ -17,16 +17,18 @@ This integration supports Admin users who have an Adobe ID and who want to strea
 The administrator who configures this integration needs the following credentials during module enablement:
 
 * Organization ID (obtained from [Adobe Admin Console](https://adminconsole.adobe.com/)), which must be at least 24 characters in length. The authenticated user must belong to this IMS organization.
+* 2FA should be enforced on the Organization level in Adobe Admin Console to enable the module.
```

Comment:
2FA should be enforced on the organization level in the Adobe Admin Console when enabling the module. See [Two-step verification](https://helpx.adobe.com/enterprise/using/authentication-settings.html#two-step-verification).

Reply:
tweaked this one line and added a link